server/app/services/export/CsvColumnFactory.java

```diff
@@ -0,0 +1,490 @@
+            .setColumnType(columnType)
+            .setHeader(formatHeader(q.getLine2Path()))
+            .setQuestionPath(q.getContextualizedPath())
+            .setAnswerExtractor(aq -> ((AddressQuestion) aq).getLine2Value().orElse(""))
+            .build(),
+        Column.builder()
+            .setColumnType(columnType)
+            .setHeader(formatHeader(q.getCityPath()))
+            .setQuestionPath(q.getContextualizedPath())
+            .setAnswerExtractor(aq -> ((AddressQuestion) aq).getCityValue().orElse(""))
+            .build(),
+        Column.builder()
+            .setColumnType(columnType)
+            .setHeader(formatHeader(q.getStatePath()))
+            .setQuestionPath(q.getContextualizedPath())
+            .setAnswerExtractor(aq -> ((AddressQuestion) aq).getStateValue().orElse(""))
+            .build(),
+        Column.builder()
+            .setColumnType(columnType)
+            .setHeader(formatHeader(q.getZipPath()))
+            .setQuestionPath(q.getContextualizedPath())
+            .setAnswerExtractor(aq -> ((AddressQuestion) aq).getZipValue().orElse(""))
+            .build(),
+        // Address correction columns
+        Column.builder()
+            .setColumnType(columnType)
+            .setHeader(formatHeader(q.getCorrectedPath()))
+            .setQuestionPath(q.getContextualizedPath())
+            .setAnswerExtractor(aq -> ((AddressQuestion) aq).getCorrectedValue().orElse(""))
+            .build(),
+        Column.builder()
+            .setColumnType(columnType)
+            .setHeader(formatHeader(q.getLatitudePath()))
+            .setQuestionPath(q.getContextualizedPath())
+            .setAnswerExtractor(
+                aq ->
+                    ((AddressQuestion) aq)
+                        .getLatitudeValue()
+                        .map(ExportFormatUtils::formatLatOrLongAsString)
+                        .orElse(""))
+            .build(),
+        Column.builder()
+            .setColumnType(columnType)
+            .setHeader(formatHeader(q.getLongitudePath()))
+            .setQuestionPath(q.getContextualizedPath())
+            .setAnswerExtractor(
+                aq ->
+                    ((AddressQuestion) aq)
+                        .getLongitudeValue()
+                        .map(ExportFormatUtils::formatLatOrLongAsString)
+                        .orElse(""))
+            .build(),
+        Column.builder()
+            .setColumnType(columnType)
+            .setHeader(formatHeader(q.getWellKnownIdPath()))
+            .setQuestionPath(q.getContextualizedPath())
+            .setAnswerExtractor(
+                aq ->
+                    ((AddressQuestion) aq)
+                        .getWellKnownIdValue()
+                        .map(w -> Long.toString(w))
+                        .orElse(""))
+            .build(),
+        Column.builder()
+            .setColumnType(columnType)
+            .setHeader(formatHeader(q.getServiceAreasPath()))
```

Comment:
@gwendolyngoetz I used getServicesAreasPath() here cause that's what is returned by getAllPaths(), which is was was called before.